pyproject.toml

```diff
@@ -42,6 +45,7 @@ dev = [
     "types-psutil>=6.0.0.20240621",
     "requests>=2.32.3",
     "types-requests>=2.32.0.20240712",
+    "fastapi>=0.112.0",
```

Comment:
Do we need this dependency?

Reply:
It's a dev dependency, needed mainly to type-check the sample app (and verify that the DBOS and fastapi types play well together).